pkg/npm/http-api/example/index.html

```diff
@@ -1,98 +1,114 @@
     <title>Demo</title>
-    <script src="browser.js"></script>
+    <script src="../dist/urbit-http-api.min.js"></script>
     <style>
-        @import url("https://rsms.me/inter/inter.css");
-        @font-face {
-            font-family: "Source Code Pro";
-            src: url("https://storage.googleapis.com/media.urbit.org/fonts/scp-regular.woff");
-            font-weight: 400;
-        }
-        body {
-            margin: 0 auto;
-            max-width: 70ch;
-            padding: 2ch;
-            font-family: 'Inter', sans-serif;
-        }
-        #mylog {
-            white-space: pre-wrap;
-            padding: 2ch;
-            background: black;
-            color: white;
-            font-family: 'Source Code Pro', monospace;
-        }
-        #mylog div {
-            margin-bottom: 1rem;
-        }
-        .chunk {
-            border-bottom: 1px dashed currentColor;
-        }
+      @import url('https://rsms.me/inter/inter.css');
+      @font-face {
+        font-family: 'Source Code Pro';
+        src: url('https://storage.googleapis.com/media.urbit.org/fonts/scp-regular.woff');
+        font-weight: 400;
+      }
+      body {
+        margin: 0 auto;
+        max-width: 70ch;
+        padding: 2ch;
+        font-family: 'Inter', sans-serif;
+      }
+      #mylog {
+        white-space: pre-wrap;
+        padding: 2ch;
+        background: black;
+        color: white;
+        font-family: 'Source Code Pro', monospace;
+      }
+      #mylog div {
+        margin-bottom: 1rem;
+      }
+      .chunk {
+        border-bottom: 1px dashed currentColor;
+      }
     </style>
-</head>
-<body>
+  </head>
+  <body>
     <details>
-        <summary>Show instructions</summary>
-    <p>Assuming you are running a fakezod on port 8080, run</p>
-    <code id="instructions">|cors-approve '{window.location.origin}'</code>
-    <p>in its dojo.</p>
-    <p>Press the button to run the code below. Output will be logged. You should see <code>&lt; ~zod: opening airlock</code> in your dojo.</code> Create a chat and send a message to see the events logged.</p>
-    <pre>window.airlock = await Urbit.authenticate({
+      <summary>Show instructions</summary>
+      <p>Assuming you are running a fakezod on port 8080, run</p>
+      <code id="instructions">|cors-approve '{window.location.origin}'</code>
+      <p>in its dojo.</p>
+      <p>
+        Press the button to run the code below. Output will be logged. You
+        should see <code>&lt; ~zod: opening airlock</code> in your dojo. Create
+        a chat and send a message to see the events logged.
+      </p>
+      <pre>
+window.airlock = await UrbitHttpApi.Urbit.authenticate({
     ship: 'zod',
     url: 'localhost:8080',
     code: 'lidlut-tabwed-pillex-ridrup',
     verbose: true
 });
-window.airlock.subscribe('chat-view', '/primary', { event: console.log });</pre>
+window.airlock.subscribe('chat-view', '/primary', { event: console.log });</pre
```

Comment:
Should be changed to `('graph-store', '/updates', { event: console.log })`, also I'm curious as to whether this actually works bc this might not be set correctly

Reply:
EDIT: nvm, saw the duck punch